### DISCUSSION TITLE
How to release multiple applications in same repository

### DISCUSSION THREAD

**Discussion by rtfpessoa** [goreleaser/goreleaser#1926] [Q&A] [answered]
I need to release multiple applications in a monorepo which means versions will colide.

From what I looked people usually go with prefixes like `api/1.0.0`.

I cannot seem to find a way to accomplish this on goreleaser.

I found that @caarlos0 did a poll on twitter some years ago https://twitter.com/oieduardorabelo/status/1032239539547303936 and that #413 was closed. Anyone knows if there is a way to go around this?

**Comment by caarlos0** [comment, marked as answer]
We don't support it yet, no.

There was a PR about it but it was never merged https://github.com/goreleaser/goreleaser/pull/723

and it is still listed in the "road to v1" issue: https://github.com/goreleaser/goreleaser/issues/807

**Reply by rtfpessoa** [reply]
Nice to know. Anything I can help with?
